index.html

```diff
@@ -0,0 +1,228 @@
+                    <div>
+                        <h3 class="bold speaker-name"></h3>
+                        <p class="orange-color bold speaker-job"><p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li>
+                            <img src="images/speaker-2.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-3.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                        <li class="hide-on-mobile">
+                            <img src="images/speaker-4.png" alt="speaker number
+                                one">
+                            <div>
+                                <h3 class="bold speaker-name"></h3>
+                                <p class="orange-color bold speaker-job"></p>
+                                <hr class="speaker-hr">
+                                <p class="speaker-graduated"></p>
+                            </div>
+                        </li>
+                    </ul>
+                    <div id="more-button-wrapper">
+                        <button>More <i class="bi bi-chevron-down orange-color"></i></button>
+                    </div>
+                </section>
+
+                <section id="partners">
+                    <h1>Partners</h1>
+                    <hr class="short-orange-hr">
+                    <div id="partner-items-wrapper">
+                        <ul>
+                            <li>Google</li>
+                            <li>Facebook</li>
+                            <li>Uber</li>
+                            <li class="last-tow-partners">airbnb</li>
+                            <li class="last-tow-partners">Naver</li>
+                        </ul>
+                    </div>
+                </section>
+
+                <footer>
+                    <h4>Dev Courses 2021</h4>
+                    <p>2021 Dev Courses<br>
+                        All Rights Reserved</p>
+                </footer>
```

Comment:
- [ ] Your _footer_ section does not follow the provided template design. Your footer should:
- contain a logo aligned to the left and text aligned to the right.
- have a white background for mobile size.
- have a dark background for desktop size

Kindly ensure to improve upon the above for your _footer_ section

**Your footer section:**
![image](https://user-images.githubusercontent.com/12745474/132549020-a53353f9-801c-4cc7-815a-cc2ad4c078a0.png)

**The original footer section:**
![image](https://user-images.githubusercontent.com/12745474/132549192-88b2027c-690a-489e-89cf-01c136b0bd19.png)